test/unit/BFactory.t.sol

```diff
@@ -69,7 +70,8 @@ contract BFactory_Unit_NewBPool is Base {
    * @notice Test that msg.sender is set as the controller
    */
   function test_Set_Controller(address _randomCaller) public {
-    vm.assume(_randomCaller != VM_ADDRESS);
```

Comment:
were you really hitting the cheatcode contract @wei3erHase ? It should be excluded from the fuzz dictionnary iirc

Reply:
we were in some occasions, yes, is there a way of enforcing the exclusion?